sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -390,21 +393,43 @@ def test_model_pipeline_same_as_normalize_true(LinearModel, params):
 # FIXME: 'normalize' to be removed in 1.2
 @pytest.mark.filterwarnings("ignore:'normalize' was deprecated")
 @pytest.mark.parametrize(
-    "estimator, is_sparse, with_mean",
-    [(LinearRegression, True, False),
-     (LinearRegression, False, True),
-     (LinearRegression, False, False)]
+    "estimator, params",
+    [
+     (Lasso, {"tol": 1e-16, "alpha": 0.1}),
+     (RidgeClassifier, {"solver": 'sparse_cg', "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
+     (Ridge, {"solver": 'sparse_cg', 'tol': 1e-12, "alpha": 0.1}),
+     (LinearRegression, {}),
+     ]
+)
+@pytest.mark.parametrize(
+    "is_sparse, with_mean", [
+        (False, True),
+        (False, False),
+        (True, False)
+        # No need to test sparse and with_mean=True
+    ]
 )
 def test_linear_model_sample_weights_normalize_in_pipeline(
-        estimator, is_sparse, with_mean
+        with_mean, is_sparse, estimator, params
 ):
     # Test that the results for running linear regression LinearRegression with
```

Comment:
It seems that we test more than LinearRegression now. Could you update the description

Reply:
Sure, there was also a suggestion to more this common function to test_common.py, however then we loose the diff. If you that's ok, I could move it just before merging?

Reply:
Let's just rename for the moment and merge to fix the issue. The refactor moving test would be easier to handle in a separate PR.